modules/cnf-performing-end-to-end-tests-disconnected-mode.adoc

```diff
@@ -23,7 +23,7 @@ A `mirror` executable is shipped in the image to provide the input required by `
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
-registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
+registry.redhat.io/openshift4/cnf-tests-rhel9:{product-version} \
```

Comment:
the tag is with "v" prefix, for example:
`registry.redhat.io/openshift4/cnf-tests-rhel9:v4.19`

Reply:
Apologies, I thought I had included that. Now, it's updated!